.env.client-public

```diff
@@ -0,0 +1,26 @@
+# General
+NODE_ENV=development
+BRFENERGI_LANG=sv
+PORT=3000
+
+# Services
+MAPBOX_ACCESS_TOKEN=<MAPBOX_TOKEN>
+MAPBOX_STYLE=mapbox://styles/tornqvist/ciye3wh1c000s2sqyhepqpnrz
+GOOGLE_ANALYTICS_ID=<ANALYTICS_ID>
+
+# OAuth
+BRFENERGI_SERVICE_URL=http://localhost:3000
+BRFENERGI_SESSION_SECRET=<SESSION_SECRET>
+
+# Metry credentials
+METRY_CLIENT_ID=<CLIENT_ID>
+METRY_CLIENT_SECRET=<CLIENT_SECRET>
+METRY_OPEN_CHANNEL=<OPEN_CHANNEL_ID>
+METRY_ENDPOINT=https://app.metry.io/api/v2/
+# METRY_ENDPOINT=https://metry-development.herokuapp.com/api/v2/
+
+# CMS
+PRISMIC_API=https://brf-energi.cdn.prismic.io/api
+
+# Database
+MONGO_URL=mongodb://localhost:27017/brf-energi-dev
```

Comment:
`MONGO_URL ` is only used on the server and doesn't need to be included in the client.